src/io/controls.cpp

```diff
@@ -137,7 +148,7 @@
     #define DEFAULT_BUTTON_STATS           (10)
     #define DEFAULT_BUTTON_PAUSE           (11)
     #define DEFAULT_BUTTON_YES             (-1)
-    #define DEFAULT_BUTTON_NO              (-1)
+    #define DEFAULT_BUTTON_NO              (-1)	
```

Comment:
Typo?

Reply:
Line `#define DEFAULT_BUTTON_NO              (-1)	` must be a typo, accidentally introduced a `tab`.